src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -26,6 +26,9 @@ public void toModelType_typicalPatientsFile_success() throws Exception {
                 JsonSerializableAddressBook.class).get();
         AddressBook addressBookFromFile = dataFromFile.toModelType();
         AddressBook typicalPatientsAddressBook = TypicalPatients.getTypicalAddressBook();
+        System.out.println(addressBookFromFile);
+        System.out.println("-----------------------");
+        System.out.println(typicalPatientsAddressBook);
```

Comment:
Are these comments for debugging purposes or intended? If its intended then its fine!!

Reply:
It's for debugging as I modified the JSON file